tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/division_signpost_block.html

```diff
@@ -0,0 +1,27 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+<div class="grid division-signpost">
+    <div class="grid__division-signpost">
+        {# Section heading #}
+        {% if value.title %}
```

Comment:
Update: both this and the heading for each card should be required.

Reply:
the title for the card is indeed required, but the title for the block is optional and likely won't be used

Reply:
Noted - @SharmaineLim updated me about that on the other pr.

Reply:
The section heading isn't really on the design that's why I made it optional; I added it for flexibility in case we want a heading in the future.

For the card, I had set it up to get the title from the linked page if no heading is given. (My plan was to do the same for description & image, but it turns out we don't have listing summaries & listing images on this project.)

Reply:
Ah, I've been beaten to writing the comment 😄